bciers/apps/administration/app/data/jsonSchema/operationInformation.ts

```diff
@@ -2,303 +2,374 @@ import provinceOptions from "@bciers/data/provinces.json";
-    },
-    operation_representative: {
-      type: "string",
-      title: "Operation Representative",
-    },
-  },
-};
+export const createOperationInformationSchema =
+  async (): Promise<RJSFSchema> => {
+    const naicsCodes = await getNaicsCodes();
+    const reportingActivities = await getReportingActivities();
+    const operationInformationSchema: RJSFSchema = {
+      title: "Operation Information",
+      type: "object",
+      required: [
+        "name",
+        "type",
+        "naics_code_id",
+        "activities",
+        "boundary_map",
+        "process_flow_diagram",
+      ],
+      properties: {
+        name: { type: "string", title: "Operation Name" },
+        type: {
+          type: "string",
+          title: "Operation Type",
+          enum: ["Single Facility Operation", "Linear Facility Operation"],
+        },
+        naics_code_id: {
+          type: "number",
+          title: "Primary NAICS Code",
+          anyOf: naicsCodes.map(
+            (code: {
+              id: number;
+              naics_code: string;
+              naics_description: string;
+            }) => ({
+              const: code?.id,
+              title: `${code?.naics_code} - ${code?.naics_description}`,
+            }),
+          ),
+        },
+        secondary_naics_code_id: {
+          type: "number",
+          title: "Secondary NAICS Code",
+          anyOf: naicsCodes.map(
+            (code: {
+              id: number;
+              naics_code: string;
+              naics_description: string;
+            }) => ({
+              const: code?.id,
+              title: `${code?.naics_code} - ${code?.naics_description}`,
+            }),
+          ),
+        },
+        tertiary_naics_code_id: {
+          type: "number",
+          title: "Tertiary NAICS Code",
+          anyOf: naicsCodes.map(
+            (code: {
+              id: number;
+              naics_code: string;
+              naics_description: string;
+            }) => ({
+              const: code?.id,
+              title: `${code?.naics_code} - ${code?.naics_description}`,
+            }),
+          ),
+        },
 
-const section2Schema: RJSFSchema = {
-  title: "Multiple Operators Information",
-  type: "object",
-  properties: {
-    operation_has_multiple_operators: {
-      type: "boolean",
-      title: "Does the operation have multiple operators?",
-      default: false,
-    },
-  },
-  dependencies: {
-    operation_has_multiple_operators: {
-      oneOf: [
-        {
-          properties: {
-            operation_has_multiple_operators: {
-              enum: [false],
-            },
+        activities: {
+          type: "array",
+          items: {
+            type: "number",
+            enum: reportingActivities.map(
+              (activity: { id: number; applicable_to: string; name: string }) =>
+                activity.id,
+            ),
+            // enumNames is a non-standard field required for the MultiSelectWidget
```

Comment:
The MultistepWidget requires enumNames, but it's a non-standard field that could be [deprecated in the next release](In your JSON Schema, you may also specify enumNames, a non-standard field which RJSF can use to label an enumeration. This behavior is deprecated and may be removed in a future major release of RJSF.)
Proposed tech debt card: https://github.com/orgs/bcgov/projects/122/views/2?filterQuery=purpose+&pane=issue&itemId=79158916

Reply:
Hmm I wonder if we should tackle this sooner than later. I've thought we should standardize how we do some of these since we have `enum/enumNames` and `title/const`.

Reply:
I think the SelectWidget also needs enum, not title/const